README.pod

```diff
@@ -169,47 +299,64 @@ Currently the following backends are included:
 Supports basic CRUD operations and prepared statements with placeholders
 
     my $dbh = DBIish.connect('Pg', :host<db01.yourdomain.com>, :port(5432),
-            :database<blerg>, :user<myuser>, :$password);
+            :database<blerg>, :user<myuser>, password => get-secret());
 
 Pg supports the following named arguments:
 C<host>, C<hostaddr>, C<port>, C<database> (or its alias C<dbname>), C<user>, C<password>,
 C<connect-timeout>, C<client-encoding>, C<options>, C<application-name>, C<keepalives>,
 C<keepalives-idle>, C<keepalives-interval>, C<sslmode>, C<requiressl>, C<sslcert>, C<sslkey>,
 C<sslrootcert>, C<sslcrl>, C<requirepeer>, C<krbsrvname>, C<gsslib>, and C<service>.
 
-See your PostgreSQL documentation for details.
+See your L<PostgreSQL documentation|https://www.postgresql.org/docs/current/libpq-envars.html> for details.
+
+=head3 Parameter Substitution
 
-Pg arrays are supported when fetching array fields with C<row/allrows>. You will
-get the properly typed array according to the field type.
+In addition to the C<?> style of parameter substitution supported by all drivers, PostgreSQL also supports numbered
+parameter. The advantage is that a numbered parameter may be reused
+
+   $dbh.execute('INSERT INTO tab VALUES ($1, $2, $2 - $1)', $var1, $var2);
+
+This is equivalent to the below statement except the subtraction operation is performed by PostgreSQL:
+
+   $dbh.execute('INSERT INTO tab VALUES (?, ?, ?)', $var1, $var2, $var2 - $var1);
+
+=head3 pg arrays
+
+Pg arrays are supported for both writing via execute and retrieval via C<row/allrows>.
+You will get the properly typed array according to the field type.
 
 Passing an array to C<execute> is now implemented. But you can also use the
 C<pg-array-str> method on your Pg StatementHandle to convert an Array to a
 string Pg can understand:
 
   # Insert an array via an execute statement
-  my $sth = $db.execute('INSERT INTO tab (array_column) VALUES ($1);', @data);
+  my $sth = $dbg.execute('INSERT INTO tab (array_column) VALUES ($1);', @data);
```

Comment:
Should be `$dbh`

Reply:
Thank-you for the review. I've cleaned these up as directed.